changelog/2025-06-03-dropbox-connector-0-1-0.mdx

```diff
@@ -0,0 +1,28 @@
+---
+slug: '2025-06-03-dropbox-connector-0-3-0'
+title: Conduit Dropbox Connector v0.1.0
+draft: false
+toc_max_heading_level: 4
+tags: [conduit, connector, dropbox, release, connector-release]
+---
+
+We're stepping into June with a new connector, this time
+for [Dropbox](https://www.dropbox.com). 
+
+<!--truncate-->
+
+### Key Highlights
+
+- We've developed a source and a destination connector.
+- The source and destination connectors support chunking of large files. 
+- Chunking is done using standard OpenCDC metadata. That means that the Dropbox
+  can send large files to other similar connectors (such as the upcoming Box
+  connector). It works the opposite way too!
+- You can download the
+  connector [here](https://github.com/conduitio-labs/conduit-connector-dropbox/releases/tag/v0.1.0).
+
+---
+
+Have questions or feedback? Join us on [Discord](https://discord.meroxa.com/).
```

Comment:
@hariso I just generated the new page for the dropbox connector https://github.com/ConduitIO/conduit-site/pull/311/commits/589f78dbf681b0c14f4d79735f6c0d5f76336791

We could even link to the page directly on this changelog.